client/src/redux/reducers/allocations.reducers.js

```diff
@@ -97,6 +119,32 @@ export function allocations(state = initialState, action) {
         ...action.payload
       };
     }
+    case 'ALLOCATION_OPERATION_ADD_USER_INIT': {
+      return {
+        ...state,
+        addingUserOperation: {
+          userName: '',
+          error: false,
+          loading: true
+        }
+      };
+    }
+    case 'ALLOCATION_OPERATION_ADD_USER_COMPLETE': {
+      return {
+        ...state,
+        addingUserOperation: {
+          userName: '',
+          error: false,
+          loading: false
+        }
+      };
+    }
+    case 'ALLOCATION_OPERATION_ADD_USER_ERROR': {
+      return {
+        ...state,
+        ...action.payload
+      };
+    }
```

Comment:
It's more descriptive to be explicit with how we're manipulating the state in the reducer, maybe change it to something like:
```suggestion
    case 'ALLOCATION_OPERATION_ADD_USER_ERROR': {
      return {
        ...state,
          addingUserOperation: {
            loading: false,
            error: action.payload.error,
            userName: action.payload.id
          }
      }
      };
    }
```